src/sentry/static/sentry/app/stores/hookStore.jsx

```diff
@@ -36,7 +36,7 @@ const HookStore = Reflux.createStore({
       this.hooks[hookName] = [];
     }
     this.hooks[hookName].push(callback);
-    this.trigger(hookName, [callback]);
+    this.trigger(hookName, this.hooks[hookName]);
```

Comment:
Change this to be consistent with `remove()`, this also makes it more reliable for listeners of `HookStore` to maintain state of the current hooks.

There's only one component that uses `Reflux.listenTo`, and that's been updated to support this.